source/configuration_manual/shared_mailboxes/cluster_setup.rst

```diff
@@ -58,15 +65,20 @@ in this mailbox ``someone1`` would access a mailbox with the name
 .. note:: See :ref:`user_shared_mailboxes_vs` for an explanation more details on the used variables.
 
 Additionally imapc must be configured accordingly on the backends:
- * :ref:`setting-imapc_master_user` must be set to ``%u`` e.g. the logged in
-   user becomes imapc master user
+ * :ref:`setting-imapc_master_user` must be set to ``%{userdb:sharing_user}``
+   this requires to return to the ``sharing_user`` field for backend userdb
+   lookups and thereby use it as imapc_master_user=%{userdb:sharing_user}.
+   This makes sure the ``imapc_master_user`` is not overwritten by other userdb
+   lookups which may happen on proxies or directors. In case of a simple setup
+   imapc_master_user can also be just set to `%u` (the logged in user).
  * :ref:`setting-imapc_password` must be set to the master password which is
    configured on all backends and directors
  * :ref:`setting-imapc_host` must point to a load balancer's address that
    connects to Dovecot director/proxy
  * :ref:`setting-imapc_features`: To be able to pass through ``SEARCH`` commands
-   to the sharing user's backend, :ref:`setting-imapc_features` should at least
-   contain ``search``.
+   to the sharing user's backend, access ``MODSEQ`` and ``HIGHESTMODSEQ`` and
+   proxy ACL commands to a remote backend. :ref:`setting-imapc_features`
+   should at least contain the ``search modseq acl`` features.
```

Comment:
It should actually contain much more.. I think the whole sentence could be just simplified to:

```
 * :ref:`setting-imapc_features`: For the best performance and functionality, the setting should contain
    at least ``fetch-bodystructure fetch-headers rfc822.size search modseq acl``
```